src/App.jsx

```diff
@@ -32,6 +33,7 @@ function App() {
           />
           <Route path="EazyCard" element={<EazyCard />} />
           <Route path="Food" element={<Food />} />
+          <Route path="Food/Profile" element={<FoodProfile />} />
           <Route path="Office" element={<Office />} />
```

Comment:
the path's first letter here is capitalized and the others are not ! try to do the same everywhere plz.